src/frontend/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "lanebreach-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "webpack-dev-server --open --mode development --config webpack/frontend.dev.js",
+    "build": "webpack --config webpack/frontend.prod.js --mode production"
+  },
+  "author": "Timothy Shoaf <tim@timshoaf.dev>",
```

Comment:
You don't add an author in `src/package.json` -- I'd recommend making all of the authors `Lane Breach <bikelanessf@gmail.com>`

Reply:
npm init configuration on my different machines, my bad, will change and make consistent with that email.